gui/src/components/IfEnabled.svelte

```diff
@@ -0,0 +1,15 @@
+<script lang="ts">
+  import { api } from '../lib/api';
+  import { onMount } from 'svelte';
+
+  export let feature: string;
+
+  let enabled: boolean;
+  onMount(async () => {
+    enabled = await api.kernel.isEnabled(feature);
```

Comment:
This queries the server every time the component mounts, but since the server is local, it should not be a problem for now.

Reply:
I don't love this, as it will create some flicker/pop-in that we're going to just get used to & then won't notice other flicker or pop-in until something ships.

Is there some way we can make these feature flags more static, like the `import.meta.env` stuff?

Reply:
Yeah, I'll make this better.